lambda/src/index.js

```diff
@@ -171,9 +171,9 @@ async function updateRunner (deploymentId, combinedRunner, event, error) {
   }
 }
 
-function generateEnvVars() {
+function generateEnvVars () {
   const envVarsArray = []
-  const parsedVars = JSON.parse(Buffer.from(process.env.TEST_ENV_VAR_OVERRIDES, 'base64'))
+  const parsedVars = JSON.parse(process.env.TEST_ENV_VAR_OVERRIDES, 'base64')
```

Comment:
I'm guessing you don't need to parse base64 now, right?

Reply:
yes